virtcontainers/mount.go

```diff
@@ -29,6 +29,18 @@ import (
 
 var rootfsDir = "rootfs"
 
+var systemMountPrefixes = []string{"/proc", "/dev", "/sys"}
```

Comment:
We should only constraint this list to the devices.

Reply:
Why just /dev? We cannot share /proc and /sys between the guest and host as well.

Reply:
Yes sure, but those are already handled by the fact that we won't try to bind mount them.
I guess either way, that's fine. But I want to see the CI results first

Reply:
So this is the error message I am seeing 
```
Apr 16 21:02:22 jenkins-azure-vm-ubuntu-16-04-templatee71490 crio[68573]: time="2018-04-16 21:02:22.686454249Z" level=error msg="Container creation error: rpc error: code = Internal desc = Could not run process: container_linux.go:348: starting container process caused "process_linux.go:402: container init caused \"rootfs_linux.go:58: mounting \\\"/var/lib/kubelet/pods/1f253791-41b9-11e8-b40a-000d3afd4c4f/containers/kubedns/fc4f90c8\\\" to rootfs \\\"/run/kata-containers/shared/containers/31d498c2c53faf8fe556eb7ebc30ce91ccedb9a423e71205808b88a425a9c217/rootfs\\\" at \\\"/dev/termination-log\\\" caused \\\"stat /var/lib/kubelet/pods/1f253791-41b9-11e8-b40a-000d3afd4c4f/containers/kubedns/fc4f90c8: no such file or directory\\\"\""
```

Reply:
k8s requires the termination log and it by default resides under `/dev`.

Reply:
I do not like log files living under /dev. I think a way to handle this is not to ignore all files under /dev,  but ignore device files and pass regular files to the container as 9p shared mounts.
@bergwolf @sboeuf What do you think?

Reply:
I agree with @bergwolf that we want to support it. That being said @bergwolf , how do you think we should handle the support of something like `-v /dev/gpus/:/dev/gpus/` where the directory `/dev/gpus/` contains a list of graphics devices ? Do you think we should parse the whole directory and add one by one every device (`card0`, `card1`, `card2`, ...) to the container ? And maybe also rely on a blacklist of paths that should not be passed from the host to the guest ?